docs/source/datasources/chirps.rst

```diff
@@ -0,0 +1,124 @@
+CHIRPS
+===
+
+Background
+----------
+
+The Climate Hazards Center at UC Santa Barbara (`CHC <https://www.chc.ucsb.edu/>`_),
+in collaboration with scientists at the USGS Earth Resources Observation and Science
+(`EROS <https://www.usgs.gov/centers/eros>`_) Center, created the Climate Hazards group
+Infrared Precipitation with Stations `CHIRPS <https://www.chc.ucsb.edu/data/chirps>`_
+environmental record.
+
+CHIRPS is a quasi-global (50&deg;S-50°N), high resolution (0.05°), daily, pentadal,
```

Comment:
Larger than this comment, but in that case I think it would be good to understand if the pentaal and monthy data are anything else than just the sum of the daily data over the respective time period.

Reply:
I did a couple of checks and it seems that monthly data are indeed the sum of daily data